### PR TITLE
Homework 2. Functions

### PR DESCRIPTION
**Pull request by Natali-Pchelkina**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -39,4 +39,39 @@ for(let i = 0; i < arr11.length; ++i) {
     if(!(arr11[i] % 2)) {
         console.log(arr11[i]);
     }
-}
\ No newline at end of file
+}
+//Lesson-3
+//Task 5
+let palidrome = (str) => {
```
Comment (ghost) [inline main.js]: Use `const` for function.

File: main.js [hunk 2]
```diff
@@ -39,4 +39,39 @@ for(let i = 0; i < arr11.length; ++i) {
...
+    let strLen = str.lenght;
```
Comment (ghost) [inline main.js]: What sense of that variable?

File: main.js [hunk 3]
```diff
@@ -39,4 +39,39 @@ for(let i = 0; i < arr11.length; ++i) {
...
+    if(a > b) {
+        return a
+    }else {
+        return b
+    }
+}
+console.log(max(52,21));
+
+//Task 7
+let zero = [20,33,30,48,50,88,75,69,10,100];
```
Comment (ghost) [inline main.js]: Use const.

File: main.js (hunk 1 above)
Comment (Natali-Pchelkina) [reply main.js]: replaced with const

File: main.js (hunk 2 above)
Comment (Natali-Pchelkina) [reply main.js]: delated

File: main.js (hunk 3 above)
Comment (Natali-Pchelkina) [reply main.js]: replaced with const

**Pull request merged**